packages/plugin-runner-eslint/__tests__/__snapshots__/default.test.ts.snap

```diff
@@ -16,7 +16,18 @@ Object {
       "args": Array [
         "
 /test_path/test/index.js
+  1:13  error  Insert \`;⏎\`        prettier/prettier
```

Comment:
Does this change in snapshot makes sense @beshanoe ? Seems like the unit test is using the root eslint because `prettier/prettier` is not on the fixtures, so anytime we change the rules of the eslint for garment project this snapshot may fail.

```  const result = await executeRunner2(
    handler,
    {},
    {
      cwd,
      files
    }
  );```

Is it related to this function in the tests?

Reply:
we can review this later

Reply:
Okay thanks 🙏 
Do you want me to create an issue about this?

Reply:
that would be nice so that we don't forget it, thanks!